longrunning/longrunning.go

```diff
@@ -76,9 +78,10 @@ func (op *Operation) Done() bool {
 
 // Metadata unmarshals op's metadata into meta.
 // If op does not contain any metadata, Metadata returns ErrNoMetadata and meta is unmodified.
-func (op *Operation) Metadata(meta proto.Message) error {
+func (op *Operation) Metadata(meta protoiface.MessageV1) error {
```

Comment:
Maybe use this `protoadapt` type? https://pkg.go.dev/google.golang.org/protobuf/protoadapt#MessageV1

Seems reasonable to use and is `type MessageV1 = protoiface.MessageV1`! Just what you need, no?

Reply:
perfect, thanks